lib/Runtime/Types/CMakeLists.txt

```diff
@@ -11,17 +8,17 @@ add_library (Chakra.Runtime.Types
     DynamicObjectSnapshotEnumerator.cpp
     DynamicObjectSnapshotEnumeratorWPCache.cpp
```

Comment:
@digitalinfinity I had tested with Ubuntu 15/Clang 3.6 . Can you please share some details?
```
./build.sh CXX=/usr/bin/clang++-3.6 CC=/usr/bin/clang-3.6
```